fnsort.py

```diff
@@ -57,10 +57,7 @@ def sort_footnotes(text):
 
     # Determine the order of the footnote-links in the text. If a link is used
     # more than once, its order is its first position.
-    order = []
-    for i in links:
-        if order.count(i[1]) == 0:
-            order.append(i[1])
+    order = [i[1] for i in links]
```

Comment:
should this account for not adding duplicates? currently it doesn't in this comprehension

Reply:
Agreed. My bad and glad for the review.
I will be adding an example markdown file with duplicate tag(s).
(I should prioritize the unit testing :grin: - especially given a situation like this that it should have caught.)

I made an update and my testing says the modified comprehension should be good now.

Thank you!